src/staging/bot.py

```diff
@@ -1514,6 +1516,14 @@ async def configure_devel_bci_project(self) -> None:
         await self._run_cmd(f"""{self._osc} meta attribute {prj_name} -a OSRT:Config --set 'main-repo = standard
 pkglistgen-archs = ppc64le s390x aarch64 x86_64'""")
 
+        await self._run_cmd(
+            f"{self._osc} meta attribute {prj_name} -a OBS:RejectRequests"
+            " --set 'This project is automatically updated from git. Please do"
+            " **not** send submit requests. Either create an issue or send a "
+            "pull request to the [bci-dockerfile-generator]"
```

Comment:
Does markdown work here?

Reply:
Not really: 
![image](https://github.com/user-attachments/assets/f1e25413-05f8-44c4-9e34-da19a8d3ea57)